lib/openstack_query/docs/user_docs/SERVERS.md

```diff
@@ -0,0 +1,95 @@
+
+
+## Chaining to
+Chaining from other `ServerQuery` requires passing `USER_QUERY` as the `query_type`
+
+| From           | Prop 1                       | Prop 2                       | Type        | Documentation        |
+|----------------|------------------------------|------------------------------|-------------|----------------------|
+| `UserQuery`    | UserProperties.USER_ID       | ServerProperties.USER_ID     | One-to-Many | [USERS.md](USERS.md) |
+| `ProjectQuery` | ProjectProperties.PROJECT_ID | ProjectProperties.PROJECT_ID | One-to-Many | [USERS.md](USERS.md) |
+| `FlavorQuery`  | FlavorProperties.FLAVOR_ID   | FlavorProperties.FLAVOR_ID   | One-to-Many | [USERS.md](USERS.md) |
+
+
+
+## run() meta-parameters
+
+`ServerQuery()` has the following meta-parameters that can be used when calling `run()` to fine-tune the query.
+
+| Parameter Definition       | Optional?            | Description                                                                                                                                                                                                                                                               |
+|----------------------------|----------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `from_projects: List[str]` | Yes                  | A list of project IDs or Names to limit search to<br/>Optional, if not given will run search in project given in clouds.yaml.<br/><br />Searching for specific projects in Openstack may not be possible without admin credentials - `as_admin` needs to be set to True ) |
+| `all_projects: Bool`       | Yes, default = False | If True, will run query on all available projects available to the current user - set in clouds.yaml. <br/><br /> Searching for specific projects in Openstack may not be possible without admin credentials - `as_admin` needs to be set to True )                       |
+| `as_admin: Bool`           | Yes, default = False | If True, will run the query as an admin - this may be required to query outside of given project context set in clouds.yaml. <br/><br /> Make sure that the clouds.yaml context user has admin privileges                                                                 |
+
+
+To query on all projects remember to call `run()` like so:
+```python
+    ServerQuery.run(as_admin=True, all_projects=True)
+```
+
+To query on specific projects (you may not have access to) call `run()` like so (with admin creds):
+```python
+    ServerQuery.run(as_admin=True, projects=["name-or-id1", "name-or-id2"])
+```
+
+**NOTE: This query will work with non-admin credentials.**
+**However, to query on all projects (or specific projects) you may need admin credentials - set by `clouds.yaml`**
```

Comment:
I think for the note we should say specifically that you can query for servers using non-admin credentials, however for specific projects or to query all projects you may need to have admin credentials?